ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-settings.md

```diff
@@ -124,7 +128,11 @@
 
             # Path-group name.
           - name: <str>
+            interfaces:
+
+                # Interface name.
+              - name: <str; required; unique>
 
-            # The public IP address of the Route Reflector for this path-group.
-            ip_address: <str>
+                # The public IP address of the Route Reflector for this path-group.
+                ip_address: <str>
```

Comment:
This could be a list. If a pathfinder has multiple interfaces that provide internet connectivity, you could have an IP per interface?

Reply:
For static tunnels, we can configure multiple public ip address on the edge side - 
peer static router-ip PathinderVteptIP
    ipv4 address ip1
    ipv4 address ip2

We'll have to change the stun client config to use the list of public ips.


On the pathfinder side, we'll have to change the stun server to bind to all the interfaces.

Reply:
Ignore this please. The schema is correct.

Reply:
schema is correct but code is not there :) adding it in next commit!

Reply:
> We'll have to change the stun client config to use the list of public ips.
Can you please share a snippet of this - I cannot configure multiple IPs under the same server-profile. Should I make one server profile per IP ?

Reply:
I see we had this in the previous implementation - 

stun
              client
                  no disabled
              % for pfrId in device.peerPathfinders:
                  <% server = parser.allDevices[pfrId] %>
                  server-profile ${server.dpsIp}
                      port ${server.stunPort}
                  % for intfName in server.wanInterfaces:
                      % if server.getIntfPublicIp(intfName):
                      ip address ${server.getIntfPublicIp(intfName)}
                      % endif
                  %endfor
              %endfor
          !

Let me check this again.

Reply:
So the snippet above is not supported from what I can see for `stun.client`:

```
wan1-dc1(config-stun-server-profile)#show active
stun
   client
      server-profile zz
         ip address 1.1.1.1
wan1-dc1(config-stun-server-profile)#ip address 1.1.1.2
wan1-dc1(config-stun-server-profile)#show active
stun
   client
      server-profile zz
         ip address 1.1.1.2   <<<<<<<<<<<< overwritten
wan1-dc1(config-stun-client)#
wan1-dc1(config-stun-client)#server-profile zz
wan1-dc1(config-stun-server-profile)#ip address 1.1.1.2 ?
  <cr>

wan1-dc1(config-stun-server-profile)#ip address 1.1.1.2 1.1.1.1
% Invalid input.  <<<<<<< only one IP supported
```